src/pr_risk_summary.py

```
@@ -129,6 +133,56 @@ def _parse_iso(ts: Optional[str]) -> Optional[datetime]:
 
+def build_dashboard_summary(pr_rows: Iterable[Dict], *, window_days: int = 7, now_iso: Optional[str] = None) -> Dict:
+    rows = list(pr_rows)
+    now = _parse_iso(now_iso) if now_iso else datetime.now(timezone.utc)
+    if now is None:
+        now = datetime.now(timezone.utc)
+    cutoff = now - timedelta(days=window_days)
+
+    normalized = [summarize_pr_row(row) for row in rows]
+    in_window = [
+        row
+        for row, raw in zip(normalized, rows)
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)